api/Cargo.toml

```diff
@@ -5,21 +5,20 @@ authors = ["Markus Merklinger <markus.merklinger@code.berlin>"]
 edition = "2018"
 
 [dependencies]
-tokio = { version = "0.2.22", features = ["macros"] }
-warp = "0.2.5"
-futures = "0.3.5"
-hyper = "0.13"
-log = "0.4.11"
-env_logger = "0.7.1"
+chrono = { version = "0.4", features = ["serde"] }
+dotenv = "0.15"
+env_logger = "0.7"
 envconfig = "0.8"
 envconfig_derive = "0.8"
-dotenv = "0.15.0"
-serde = "1.0.116"
-serde_derive = "1.0.116"
-serde_json = "1.0.57"
-chrono = { version = "0.4.18", features = ["serde"] }
-once_cell = "1.4.1"
-tarpc = { version = "0.21.0", features = ["full"] }
+futures = "0.3"
+http = "0.2"
+hyper = "0.13"
 identity = { path = "../identity" }
-tokio-serde = { version = "0.6.1", features = ["json"] }
-http = "0.2.1"
+log = "0.4"
+once_cell = "1.4"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+tarpc = { version = "0.21", features = ["full"] }
+tokio = { version = "0.2", features = ["full"] }
+tokio-serde = { version = "0.6", features = ["json"] }
+warp = "0.2"
```

Comment:
This was also my approach in the beginning, but with the introduction of automated tests and CI/CD we figured that sometimes also the patch versions break things. Would recommend to leave it in, even though we will have more work with it.

Reply:
So far I was always not pinning the patch, unless it is breaking. But pinning by default would also work for me.